configs/mender_convert_demo_config

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Missing copyright and license

Reply:
We don't have a copyright header in any of the configuration files as of now, but maybe we should? :thinking: